src/managed/OpenLiveWriter.PostEditor/Video/WebVideoSource.cs

```diff
@@ -376,7 +376,7 @@ private string CreateEmbedHtml(string embed)
 
         private static string CreateHtml(string html, string color)
         {
-            return String.Format(CultureInfo.InvariantCulture, "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01//EN\" \"http://www.w3.org/TR/html4/strict.dtd\"><body style='margin: 0; padding: 0;background-color: #{1}'>{0}</body></html>", html, color);
+            return String.Format(CultureInfo.InvariantCulture, "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01//EN\" \"http://www.w3.org/TR/html4/strict.dtd\"><head><meta content='IE=Edge' http-equiv='X-UA-Compatible'/></head><body style='margin: 0; padding: 0;background-color: #{1}'>{0}</body></html>", html, color);
```

Comment:
@nvella, I think it would be better if we add HTML5 syntax here instead of older html4. Should I change it to HTML5 syntax?

Reply:
Sounds good to me 👍 The HTML5 syntax is much cleaner looking, and should work on all of the platforms we currently support